Sasquatch/SasquatchSwift-Preview-Extension/Info.plist

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Reuse `Sasquatch/SasquatchSwift-Extension/Info.plist`. You can set Info.plist path in Build Settings.